index.js

```diff
@@ -57,6 +57,63 @@ bot.registerCommand(
     }
 );
 
+bot.registerCommand(
+    "createPoll",
+    (msg, args) => {
+
+        const emojiSet = args[0];
+        let pollTitle = [];
+        let optionSet = [];
+        let optionBuilder = [];
+        let argCount = 0;
+
+        args.shift(); // Removes the first option that determins what emojis to use
```

Comment:
Generally, I'd avoid modifying the input parameters. It probably doesn't matter here, but it's a good habit to get into since modifying the array like this also modifies it for the caller. For example, I'd do something like 

```
const descriptionIndex = args.indexOf('Description:');
if (descriptionIndex == -1) {
  throw new Error("ahhhh");
}
const polltitle = args.slice(1, descriptionIndex).join(' ');
```